### DISCUSSION TITLE
Multi Select Pills

### DISCUSSION THREAD

**Discussion by amnesca** [livewire/flux#513] [Feature Requests] [answered]
Hi, what happened to the select.pills option that was mentioned at the [demo](https://youtu.be/31pBMi0UdYE?t=2532)? I can't seem to find anything in the docs about it. Thanks

**Comment by joshhanley** [comment]
@amnesca The multi select component got removed before v1 as Caleb wasn't happy with it. Once we're on top of bugs, I'm sure he will start looking at it again.

**Reply by beilsma** [reply]
What is the current status of this feature? The multi-select pills were my primary reason for pre-purchasing Flux after seeing them in Caleb's demo, and I was disappointed to find they weren't included in v1. We're now already in v2 and still waiting for this functionality to be implemented. I've almost completed migrating a big project to Flux but can't release it yet because I'm still waiting for this essential feature.

**Reply by mauritskorse** [reply]
Checkboxes now have a pills variant: https://fluxui.dev/components/checkbox#pills

**Reply by beilsma** [reply]
The checkbox pills work well for a limited number of options, but this becomes ineffective when dealing with many options that users need to search through efficiently.

**Reply by MCKLtech** [reply]
+1 for this feature. We have requirements for searching large data sets, and we need to do it dynamically.

**Comment by mnabialek** [comment]
Yeah, would be really good to have it back with possibility to customize look of selected pills

**Comment by Uhasith** [comment]
+1 Vote for this feature 😃

**Comment by joshhanley** [comment, marked as answer]
We've just added support for pillboxes in v2.4!

See the blog post here https://fluxui.dev/blog/2025-09-15-pillbox
And you can find the documentation here https://fluxui.dev/components/pillbox
